go/vt/vtgate/executor_dml_test.go

```diff
@@ -464,6 +464,25 @@ func TestDeleteEqual(t *testing.T) {
 	}
 }
 
+func TestUpdateScatter(t *testing.T) {
```

Comment:
Luckily, there's already a test for scatter-dmls (non-annotated) in the resharding test. However, those tests are ageing (v2). We should look at upgrading them to use v3.